CrewChiefV4/Events/Strategy.cs

```diff
@@ -457,13 +478,8 @@ private float getTimeLossEstimate(CarData.CarClass carClass, String trackName, D
                             playerTimeSpentInPitLane = timeLossEstimate - 2;
                         }
                     }
-                    Console.WriteLine("Got pitstop time loss estimate from opponent stop times - expect to lose " + timeLossEstimate + " seconds");
```

Comment:
I would keep logs unless they are spammy.

Reply:
That method is called once per car per lap in this branch, so I think it's spamming now